src/components/m-dropdown/index.js

```diff
@@ -214,6 +215,14 @@ class AXADropdown extends BaseComponentGlobal {
   get native() {
     return this.getAttribute('native');
   }
+
+  set selectedItem(value) {
+    this.setAttribute('selected-item', JSON.stringify(value));
+  }
+
+  get selectedItem() {
+    return JSON.parse(this.getAttribute('selected-item'));
```

Comment:
why setting this as prop? can it not just e a function or helper inside `index.js`?

Reply:
Well, I need info in `template.js` whether there is selected item or not. I had a function for it in `template.js` but it was suggested that I remove it and get that info from the `index.js`. 
I'm not sure which is the best way to do it, and I would appreciate your suggestion.

Reply:
i think is meant to just pass it down instead of calculating inside the template. Now we have a attribute that always changes and triggers a new-rendering. Can the funciton not just live in `index.js` and pass the filtered value with `get item()`? then in template u access it easly via wcNode

Reply:
if its not clear, we can have a quick call

Reply:
Thanks @LucaMele for the help. Requested changes implemented.